### DISCUSSION TITLE
Still needed if using STM32 extension?

### DISCUSSION THREAD

**Discussion by ganzziani** [MaJerle/stm32-cube-cmake-vscode#24] [Q&A] [answered]
Hello, thanks for writing this tutorial. I'm getting started on an STM32 project and I want to code in VS Code.
If using the STM32 extension, is this tutorial still relevant?
Is one option better than the other?

**Comment by MaJerle** [comment, marked as answer]
The tutorial here well explains the basics, what you need to use and how you can integrate it. At least you get the underlying requirements.

Then, STM32 extension is certainly good approach for you to move fast with development. My tutorial was written before ST released official extension, hence it remains public. But its concept is still more than valid.

Hope it helps.
